stable/aws-load-balancer-controller/templates/deployment.yaml

```diff
@@ -135,8 +135,17 @@ spec:
       nodeSelector:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-    {{- with .Values.affinity }}
       affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
i think it's better to be `preferredDuringSchedulingIgnoredDuringExecution`

Reply:
ack